pkg/cli/mirror/operator.go

```diff
@@ -177,8 +240,8 @@ func (o *OperatorOptions) renderDCFull(ctx context.Context, reg *containerdregis
 	ctlgRef := ctlg.Catalog //applies for all docker-v2 remote catalogs
 	if ctlg.IsFBCOCI() {
 		// initialize path where we assume the catalog config dir is <current working directory>/olm_artifacts/<repo>/<config folder>
-		ctlgRef, err = o.getOperatorCatalogRef(ctx, ctlg.Catalog)
-		if err != nil {
+		var ok bool
+		if ctlgRef, ok = o.operatorCatalogToFullArtifactPath[ctlg.Catalog]; !ok {
```

Comment:
At this point, err is not yet defined.
We should return a `fmt.Errorf(...)` in this case

Reply:
good catch! fixed in https://github.com/openshift/oc-mirror/pull/611/commits/f871dbec7e8c02f27c0bac470bb8c62d27c8faf0